analysers/analyser_osmosis_highway_deadend.py

```diff
@@ -442,8 +446,16 @@ def __init__(self, config, logger = None):
             resource = 'https://wiki.openstreetmap.org/wiki/Tag:service%3Ddrive-through')
 
         self.callback21 = lambda res: {"class": 1, "data": [self.way_full, self.node_full, self.positionAsText]}
-        self.callback22 = lambda res: {"class": 2, "data": [self.way_full, self.node_full, self.positionAsText]}
-
+        self.callback22 = lambda res: {
+            "class": 2,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": {"en": res[3]}
+        }
+        self.callback43 = lambda res: {
+            "class": 3,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": {"en": "Oneway inaccessible: {}".format(res[3])}
```

Comment:
Should either become translatable (see the function `T_(string, parameter1, parameter2, ...)`, but lets await what frodrigo says about my comment on line 452

Reply:
Not sure how to implement this.

Reply:
My own preference would be:
`"text": T_("Oneway inaccessible: {0}", res[3])`
Just to avoid confusion that something would be wrong with the highway tag.

But it might be that @frodrigo also prefers to use the same here:
`"text": {"en": res[3]}`

@frodrigo you choose? You didn't comment on this one so I'm not sure if you preferred it for both or just for the first one?